src/DependencyInjection/Configuration.php

```diff
@@ -41,6 +41,17 @@ public function getConfigTreeBuilder()
 
         $this->addBundlesSection($root);
 
+        $root
+            ->children()
+                ->arrayNode('ivory_ckeditor')
+                    ->addDefaultsIfNotSet()
+                    ->canBeEnabled()
+                    ->children()
+                        ->scalarNode('config_name')->defaultValue('cmf_sonata_phpcr_admin_integration')->end()
```

Comment:
ah, ok. can you add a phpdoc about that to the admin classes where we set the option? we also must not forget to document that. can you add an issue in symfony-cmf/symfony-cmf-documentation please?